ext/IPC-Open3/t/IPC-Open3.t

```diff
@@ -1,7 +1,7 @@
 #!./perl -w
 
 BEGIN {
-    require Config; Config->import;
+    use Config;
```

Comment:
here this is fine as not @INC is setup, but in this case the `use Config` should not occurs in BEGIN.
I think it would be a better choice to use the `@INC` setup boilerplate as performed in most tests